lib/paperclip/attachment.rb

```diff
@@ -240,7 +240,8 @@ def dirty?
     # the instance's errors and returns false, cancelling the save.
     def save
       flush_deletes unless @options[:keep_old_files]
-      if @options[:only_process].any? && !@options[:only_process].include?(:original)
+      process = only_process
```

Comment:
Maybe it would be better to memoize `only_process` method?
If it's called somewhere else, proc will be computed again. What do you think?